ui/jobs/components/drk.ts

```diff
@@ -35,7 +35,7 @@ export const setup = (bars: Bars): void => {
       }
     }
 
-    const oldSeconds = darksideBox.duration ?? 0 - darksideBox.elapsed;
+    const oldSeconds = (darksideBox.duration ?? 0) - darksideBox.elapsed;
```

Comment:
I found that this line can be 

```suggestion
    const oldSeconds = darksideBox.value;
```

Reply:
Done, but with an extra simplification too.